heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -364,7 +364,10 @@ proto::system::Status* TMaster::RegisterStMgr(
     // First check to see if that other guy has timed out
     if (!stmgrs_[stmgr_id]->TimedOut()) {
       // we reject the new guy
-      LOG(ERROR) << "Another stmgr exists with the same id and it hasn't timed out" << std::endl;
+      LOG(ERROR) << "Another stmgr exists at "
```

Comment:
Also, shall we tell everyone there is `grep -B`? What if people don't know? Do you just tell them to RTFM? Just how hard is it to add a stream id? If you do not do it, I will do it in another PR.